src/stdlib_io.f90

```diff
@@ -0,0 +1,104 @@
+module stdlib_io
+use stdlib_types
+implicit none
+private
+public loadtxt, savetxt
```

Comment:
Style question: Should we use `::` universally? I think yes, because it visually separates statement word from the listed variable and procedure names

Reply:
Yes. I would love to have some formatting capability, so that we can just run the formatter on the CI and it will fail a CI if things are not formatted correctly.

Reply:
Best thing I've found so far is `findent` and `eclint` for Fortran. `findent` works off an actual AST and claims to test against all code examples in the latest Modern Fortran Explained by MRC, so it should be robust. Probably won't help with this particular style issue, but I haven't been using it in earnest yet as I just discovered it (and added it to Homebrew).

Reply:
Fixed in 43ed8374a6038cc7f4e087c21a3faf857249f4c6.

Reply:
I created #29 for the autoformatting.